app/scripts/sentry-install.js

```diff
@@ -1,7 +1,7 @@
 import setupSentry from './lib/setupSentry';
 
 // The root compartment will populate this with hooks
-global.stateHooks = {};
+global.stateHooks = global.stateHooks || {};
```

Comment:
it is really weird that a file named `sentryInstall` is the thing that sets up "`stateHooks`". In the webpack builds initialization file this `stateHooks` object is set up there. For compatability with both build systems I added this line.

Reply:
:heavy_check_mark: